roles/brotli_nginx/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 nginx_brotli_version: v0.1.2
-nginx_modules_path: /etc/nginx/modules
+nginx_modules_path: /usr/lib/nginx/modules
```

Comment:
I don't think we want to include either of these these nginx installation changes. The original issue should be resolved by this PR: https://github.com/openfoodfoundation/ofn-install/pull/625. I manually retested it this morning on both Ubuntu 16 and 18.

Reply:
ping @kabhotz